utopia-remix/app/env.server.ts

```diff
@@ -10,27 +10,29 @@ declare global {
 
 export type AppEnv = 'local' | 'stage' | 'prod' | 'test'
 
+const environment = getAppEnv()
+console.info(`Remix app environment: ${environment}`)
+
 export const ServerEnvironment = {
-  environment: mustEnv('APP_ENV') as AppEnv,
+  environment: environment,
   // The URL of the actual backend server in the form <scheme>://<host>:<port>
-  BackendURL: mustEnv('BACKEND_URL'),
+  BACKEND_URL: mustEnvOrLocalFallback('BACKEND_URL', 'http://localhost:8002'),
   // the CORS allowed origin for incoming requests
-  CORSOrigin: mustEnv('CORS_ORIGIN'),
+  CORS_ORIGIN: mustEnvOrLocalFallback('CORS_ORIGIN', 'http://localhost:8000'),
   // Auth0 credentials
-  AUTH0_ENDPOINT: optionalEnv('AUTH0_ENDPOINT', '<AUTH0_ENDPOINT>'),
-  AUTH0_CLIENT_ID: optionalEnv('AUTH0_CLIENT_ID', '<AUTH0_CLIENT_ID>'),
-  AUTH0_REDIRECT_URI: optionalEnv('AUTH0_REDIRECT_URI', '<AUTH0_REDIRECT_URI>'),
-  // FGA Credentials
-  FGA_STORE_ID: optionalEnv('FGA_STORE_ID', ''),
-  FGA_CLIENT_ID: optionalEnv('FGA_CLIENT_ID', ''),
-  FGA_SECRET: optionalEnv('FGA_SECRET', ''),
-  FGA_API_HOST: optionalEnv('FGA_API_HOST', ''),
-  FGA_API_TOKEN_ISSUER: optionalEnv('FGA_API_TOKEN_ISSUER', ''),
-  FGA_API_AUDIENCE: optionalEnv('FGA_API_AUDIENCE', ''),
-  FGA_API_SCHEME: optionalEnv('FGA_API_SCHEME', 'https'),
```

Comment:
I figured we don't really need this - as we'll always have ssl on prod and never on local

Reply:
wait, on local sometimes you want to work against a remote fga server ('https') and sometimes against a local one ('http'). It's actually important (we can try and set it up somewhere else, but we do need it)